app/src/pages/Dashboard/Groups/Views/Assignments.vue

```diff
@@ -149,8 +158,29 @@
             this.getGroupTodos();
             this.creatingTodo = false;
           })
-
       },
+		removeTodo( itemId ) {
+
+          this.loadingTodos = true;
+
+          this.$http.delete('/wp-json/studychurch/v1/assignments/' + itemId, {
```

Comment:
Use VueX to handle this. You'll need to create a new action in the Assignment store.